app/src/main/res/layout/fragment_qr_player_profile.xml

```diff
@@ -38,7 +38,7 @@
         android:layout_height="wrap_content"
         android:padding="@dimen/large_padding"
         app:layout_constraintLeft_toLeftOf="parent"
-        android:background="@drawable/rec_frame_bg"
+        android:background="@drawable/rec_test_2"
```

Comment:
can we have a more descriptive name than "test_2"?

Reply:
those are just temporary until I figure out the transparency issue